Add `Playground` page for Lightpanda Cloud (#80)
lightpanda-io/docs: src/content/run-on-lightpanda-cloud/playground.mdx

Open [Scripts](https://console.lightpanda.io/playground/scripts) under Playground in the console and create one. The editor opens on the Playwright example below:

<Tabs items={['Playwright', 'Puppeteer']}>
  <Tabs.Tab>
```js copy
import { chromium } from 'playwright-core';

// Connect Playwright's chromium driver to the browser.
const browser = await chromium.connectOverCDP(process.env.CDP_WS_URL);

const context = await browser.newContext({});
const page = await context.newPage();

await page.goto('https://wikipedia.com/');

const title = await page.locator('h1').textContent();
console.log(title);

await page.close();
await context.close();
await browser.close();
```
  </Tabs.Tab>
  <Tabs.Tab>
```js copy
import puppeteer from 'puppeteer-core';

// Connect Puppeteer to the browser.
const browser = await puppeteer.connect({
  browserWSEndpoint: process.env.CDP_WS_URL,
});

const context = await browser.createBrowserContext();
const page = await context.newPage();

await page.goto('https://wikipedia.com/');

const title = await page.$eval('h1', el => el.textContent);
console.log(title);

await page.close();
await context.close();
await browser.disconnect();
```
  </Tabs.Tab>

> </Tabs>

Reviewer: A couple of thoughts:
- The examples use `CDP_WS_URL`, but we only explain what it is in the following section. This might be confusing 🤔 Maybe we should briefly introduce it in this section first and keep the next section to go into more details?
- Also I’m wondering whether we need this whole section as written. It mainly explains how to navigate to the `Scripts` page and what readers will see after selecting `Create a script`.

Author: The navigation instructions are gone and the console link moved to the intro. Added back a sentence on `CDP_WS_URL`. 
But I prefer keep the exemples for now, so we have both playwright and pupeteer examples
https://github.com/lightpanda-io/docs/blob/b279cacae39100892eca0ad0a6a382770bd077ec/src/content/run-on-lightpanda-cloud/playground.mdx#L12-L14

## Connect with CDP_WS_URL

> Your script never carries a token. The playground sets one environment variable, `CDP_WS_URL`, which holds the WebSocket address your client uses to reach the browser over the [Chrome DevTools Protocol](https://chromedevtools.github.io/devtools-protocol/) (CDP). The playground assembles it from the choices in the Default variables panel:

Reviewer: I feel like "Your script never carries a token" is a bit misleading since we list `Token` in the default variables below 😄 Maybe we could rephrase it to something like:

```suggestion
You do not need to hard-code a token in your script. The playground sets one environment variable, `CDP_WS_URL`, which holds the WebSocket address your client uses to reach the browser over the [Chrome DevTools Protocol](https://chromedevtools.github.io/devtools-protocol/) (CDP). The playground assembles it from the choices in the Default variables panel:
```

Author: sure. Took your wording:
https://github.com/lightpanda-io/docs/blob/b279cacae39100892eca0ad0a6a382770bd077ec/src/content/run-on-lightpanda-cloud/playground.mdx#L63

<Callout type="info">
`console.log(process.env.CDP_WS_URL)` prints `***`, not the URL. The URL carries an access token created for the run, and run output is stored in the console, so the playground masks the URL wherever it appears in output.
</Callout>

## What the runtime provides

Scripts run in an isolated container. Both CDP clients are preinstalled at fixed versions, so there is no install step:

| Package | Version |
| --- | --- |
| `playwright-core` | 1.50.0 |
| `puppeteer-core` | 24.0.0 |

The rest of the environment is fixed too:

- **Language.** JavaScript and TypeScript both run through [tsx](https://tsx.is/), so `import` syntax and top-level `await` work in either without configuration.
- **Packages.** You cannot install anything beyond the two clients above.
- **Filesystem.** Read-only, apart from a writable `/tmp`.

Every run is capped:

| Limit | Value |
| --- | --- |
| Run time | 5 minutes |
| Memory | 256 MB |
| CPU | 1 core |
| Output captured | 256 KB |
| Processes | 64 |

A run that reaches 5 minutes stops and reports `timeout running program`. Output beyond 256 KB is dropped, and the run is flagged as truncated.

> These caps apply to the script process. The browser it drives is an ordinary cloud session, so it also counts toward your [plan limits](/run-on-lightpanda-cloud/limits-and-billing).

Reviewer: using "it" two times to talk about two different subjects(the script process + the browser session) feels ambiguous

```suggestion
These caps apply to the script process. The browser session driven by the script also counts toward your [plan limits](/run-on-lightpanda-cloud/limits-and-billing).
```

Author: Agreed, applied as is.
https://github.com/lightpanda-io/docs/blob/b279cacae39100892eca0ad0a6a382770bd077ec/src/content/run-on-lightpanda-cloud/playground.mdx#L105

> ## Review past runs

Reviewer: I think "review" is not really accurate here as we don't really evaluate/assess anything 🤔 
```suggestion
## Run history
```

Author: As the section was down to one sentence, I finally decided it did not need its own heading, so the retention line now sits with the other run-level facts. Nobody needs docs to find the Runs page, it is in the console menu.
https://github.com/lightpanda-io/docs/blob/b279cacae39100892eca0ad0a6a382770bd077ec/src/content/run-on-lightpanda-cloud/playground.mdx#L107

> Every run is kept. [Runs](https://console.lightpanda.io/playground/script-runs) under Playground lists them with their state, start time, and duration, and you can filter the list by script. Open a run to read the output it produced.

Reviewer: Since runs are saved only for 30 days in our system maybe we should mention it. WDYT?

I'm not a fan of listing every visible fields  and filters. It duplicates what is already shown on the UI IMO. Also it will easily be out of sync with the real UI if we then add new fields 😄 

```suggestion
The [Runs page](https://console.lightpanda.io/playground/script-runs) shows your run history and output for the past 30 days.
```

Author: Ah ok, thanks for the precision
https://github.com/lightpanda-io/docs/blob/b279cacae39100892eca0ad0a6a382770bd077ec/src/content/run-on-lightpanda-cloud/playground.mdx#L107
